serialized-lob/src/main/java/com/iluwatar/serializedlob/Customer.java

```diff
@@ -0,0 +1,173 @@
+    var xmlOutput = new XMLOutputter();
+    return xmlOutput.outputString(element);
+  }
+
+  /**
+   * Load a element from the database.
+   *
+   * @param id1        id of the query
+   * @param dataSource the datasource
+   * @return Element
+   * @throws SQLException  if any error
+   * @throws IOException   if any error
+   * @throws JDOMException if any error
+   */
+  protected Element load(final long id1, final DataSource dataSource)
+          throws SQLException, IOException, JDOMException {
+    var sql = "select departments from customers where id = ?";
+    ResultSet resultSet = null;
+    try (var connection = dataSource.getConnection();
+         var preparedStatement =
+                 connection.prepareStatement(sql)
+    ) {
+      var result = "";
+      preparedStatement.setLong(1, id1);
+      resultSet = preparedStatement.executeQuery();
+      while (resultSet.next()) {
+        result = resultSet.getString("departments");
+      }
+      return stringToElement(result);
+    } finally {
+      if (resultSet != null) {
+        resultSet.close();
+      }
+    }
+  }
+
+  /**
+   * Insert LOB to database.
+   *
+   * @param dataSource the datasource
+   * @return the id of the inserted row
+   * @throws SQLException if any error
+   */
+  protected long insert(final DataSource dataSource) throws SQLException {
+    var sql = "insert into customers (id,name, departments) values (?,?,?)";
+    try (var connection = dataSource.getConnection();
+         var preparedStatement =
+                 connection.prepareStatement(sql)
+    ) {
+      preparedStatement.setLong(1, id);
+      preparedStatement.setString(2, name);
+      preparedStatement.setString(3,
+              elementToString(departmentsToXmlElement()));
+      preparedStatement.executeUpdate();
+      LOGGER.info(String.format("Insert: name = %s  xmlString = %s",
+              name, elementToString(departmentsToXmlElement())));
+      return id;
+    }
+  }
+
+  /**
+   * Add element to departments.
+   *
+   * @param department the department to be added
+   */
+  protected void addDepartment(final Department department) {
+    departments.add(department);
+
+  }
+
+  /**
+   * Convert filed departments to XmlElement.
+   *
+   * @return element
+   */
+  protected Element departmentsToXmlElement() {
+    var root = new Element("departmentList");
+    for (Department dep : departments) {
+      root.addContent(dep.toXmlElement());
+    }
+    return root;
+  }
+
+  /**
+   * Read element.
+   *
+   * @param source element source
+   */
+  void readDepartments(final Element source) {
+    departments.clear();
+    for (Element element : source.getChildren("department")) {
+      addDepartment(Department.readXml(element));
+    }
+  }
+
+  /**
+   * The get method of the field departments.
+   *
+   * @return departments
+   */
+  protected ArrayList<Department> getDepartments() {
+    return departments;
+  }
```

Comment:
Use Lombok to avoid boilerplate, see https://projectlombok.org/features/GetterSetter